aspnet/ajax/cdn/jquery-mobile/cdnjquerymobile110rc2.md

```diff
@@ -25,5 +25,4 @@ The jQuery Mobile 1.1.0 RC 2 library includes the following files:
 - https://ajax.aspnetcdn.com/ajax/jquery.mobile/1.1.0rc2/images/ajax-loader.png
 - https://ajax.aspnetcdn.com/ajax/jquery.mobile/1.1.0rc2/images/icons-18-black.png
 - https://ajax.aspnetcdn.com/ajax/jquery.mobile/1.1.0rc2/images/icons-18-white.png
-- https://ajax.aspnetcdn.com/ajax/jquery.mobile/1.1.0rc2/images/icons-36-black.png
-- https://ajax.aspnetcdn.com/ajax/jquery.mobile/1.1.0rc2/imagesicons-36-white.png
+- https://ajax.aspnetcdn.com/ajax/jquery.mobile/1.1.0rc2/images/icons-36-black.pn
```

Comment:
@Rick-Anderson Pretty sure this is an oopsie. Just wanted to draw your attention to it.

I'm on the validation team with Aquent doing PR reviews and just caught this. 

If there's anything I can do to make your job easier, hit me up. Easiest way to reach me is Twitter DMs... Mythoclast_BM or Teams (Braden MacBeth). 

Big fan of your work BTW!

Reply:
> @Rick-Anderson Pretty sure this is an oopsie. Just wanted to draw your attention to it.
> 
> I'm on the validation team with Aquent doing PR reviews.
> 
> If there's anything I can do to make your job easier, hit me up. Easiest way to reach me is Teams (Braden MacBeth).

You're right. Looks like https://github.com/dotnet/AspNetDocs/commit/89db037c87b02f7a9d314ea78206184eda0968b0 is the problem. 
@StephenBrentPeters can you fix that?

Reply:
See https://github.com/dotnet/AspNetDocs/pull/632, I'm reverting it.